plots/laws/thermodynamics/dependence_of_radiation_power_on_temperature.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+from sympy import solve, symbols
+from sympy.plotting import plot
+from sympy.plotting.plot import MatplotlibBackend
+from symplyphysics import print_expression
+from symplyphysics.laws.thermodynamics import stefan_boltzmann_law
+
+print(f"Formula is:\n{stefan_boltzmann_law.print_law()}")
+
+temperature_function = stefan_boltzmann_law.temperature
+
+solved = solve(stefan_boltzmann_law.law, stefan_boltzmann_law.irradiance,
+    dict=True)[0][stefan_boltzmann_law.irradiance]
+irradiance_temperature = solved.subs({
+    stefan_boltzmann_law.temperature: temperature_function,
+    stefan_boltzmann_law.units.stefan: 5.67036713*(10**(-8)),
```

Comment:
Not a good idea to hardcode numbers here. You can obtain this value from stefan-boltzmann constant using `convert_to` function.

Reply:
`units.stefan_boltzmann_constant.convert_to(units.watt/units.meter**2/units.kelvin**4)
print(units.stefan_boltzmann_constant.scale_factor)`
>> 1.85294433695108e-7*pi**5
= 5.67*10^-5
This value is not a Stefan Boltzmann constant. The Stefan Boltzmann constant has a value equal to 5.67×10^-8. I will continue to think and seek solutions.

Reply:
`>>> convert_to(units.stefan, units.watt/units.meter**2/units.kelvin**4).evalf(5)`
`5.6704e-8`

Reply:
I'm beginning to understand. Thanks!